src/node_options.h

```diff
@@ -121,6 +121,7 @@ class PerProcessOptions : public Options {
   std::string title;
   std::string trace_event_categories;
   std::string trace_event_file_pattern = "node_trace.${rotation}.log";
+  int64_t max_http_header_size = 8 * 1024;
```

Comment:
Is there a reason this can't be `uint32_t`? Avoids casts and sidesteps the issue of values < 0.